c7n/filters/core.py

```diff
@@ -422,7 +431,7 @@ def validate(self):
             if not self.data['op'] in OPERATORS:
                 raise PolicyValidationError(
                     "Invalid operator in value filter %s" % self.data)
-            if self.data['op'] == 'regex':
+            if self.data['op'] in {'regex', 'regex-cs'}:
```

Comment:
i did a quick benchmark and was surprised by the efficiency of the set here vs tuple even for low cardinalities.